yuptoo/modifiers/transform_tags.py

```diff
@@ -7,6 +7,10 @@ def run(self, host: dict, transformed_obj: dict, **kwargs):
         if tags:
             tags_modified = False
             for tag in tags:
+
+                if len(str(tag['value'])) > 255:
```

Comment:
@san7ket  - could you check `tag['value']` is of string type and then apply truncate?
Please refer - https://github.com/RedHatInsights/yuptoo/pull/48 yupana side opened pull-request for it.

Reply:
Ignore above comment as implementation changed on foreman_rh_cloud side.